### DISCUSSION TITLE
CMIP7 forcing datasets availability

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#49] [Announcements]
Some pre-CMIP7 climate forcing datasets are now available for testing, with updates and extensions compared to the CMIP6 datasets, now reaching through 2021. These datasets will be continuously updated to address any reported issues; outdated versions will be retracted, and new versions published. Once validated and refined, they will be extended for CMIP7 fast-track use. The CMIP6plus v0 forcing datasets will be available on ESGF under the input4MIPs project—use the CMIP6plus identifier to access them.

Please stay updated on the latest developments, availability, known issues, and discussions regarding the CMIP6plus and CMIP7 datasets on GitHub: [PCMDI/input4MIPs_CVs](https://github.com/PCMDI/input4MIPs_CVs).

I also recommend following—and participating in—the ongoing discussions: [PCMDI/input4MIPs_CVs Discussions](https://github.com/PCMDI/input4MIPs_CVs/discussions).

**Please note. Do not use CMIP6plus datasets for production runs!**

piControl forcing datasets can only be derived from stable historical forcing datasets. The earliest expected availability for piControl forcing datasets is late July, though some consider this timeline optimistic.

From the CMIP IPO:
Thank you for participating in the CMIP Forcings drop in session. The slides are now available at https://wcrp-cmip.org/wp-content/uploads/2024/10/2024-10-29_Forcings-drop-in_session.pdf and also on the workshop webpage within the programme with other workshop slides https://wcrp-cmip.org/event/forcings-workshop/ where the recording of the drop in will be uploaded by end of the day.

Some other useful links:

[Forcings datasets availability](https://wcrp-cmip.org/cmip7-task-teams/forcings/#forcing_datasets_availability)
[Input4MIPs ESGF Project](https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/database-views/input4MIPs_delivery-summary_CMIP6Plus.html)
[Github to feedback on the forcing datasets](https://github.com/PCMDI/input4MIPs_CVs/discussions)

**Comment by adagj** [comment]
Some relevant slides from the CMIP7 DECK Forcings suite drop in session:


![Screenshot from 2024-10-29 14-39-59](https://github.com/user-attachments/assets/1904cd76-5677-404e-ae4c-ff3406f6a0be)
![Screenshot from 2024-10-29 14-42-27](https://github.com/user-attachments/assets/1b9ecc64-172e-4252-a300-193cdb86bb08)
![Screenshot from 2024-10-29 14-43-21](https://github.com/user-attachments/assets/2f442004-8112-4d6f-8067-5df89fecc970)
![Screenshot from 2024-10-29 14-48-30](https://github.com/user-attachments/assets/6607c951-40b8-44e8-bd94-9fce308278bf)

**Comment by adagj** [comment]
From the CMIP-IPO:

A number of CMIP6Plus preliminary forcing datasets have been made [available ](https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/database-views/input4MIPs_delivery-summary_CMIP6Plus.html)via ESGF through the input4MIPs project ([https://aims2.llnl.gov/search/input4MIPs](https://aims2.llnl.gov/search?project=input4MIPs&activeFacets=%7B%22mip_era%22%3A%22CMIP6Plus%22%7D)) and are ready for testing by any interested modelling centres or data analysts.

 We encourage users to test the [available datasets ](https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/database-views/input4MIPs_delivery-summary_CMIP6Plus.html)and provide feedback via the input4MIPs_CVs repository discussions on [GitHub (https://github.com/PCMDI/input4MIPs_CVs/discussions)](https://github.com/PCMDI/input4MIPs_CVs/discussions). For further details on the CMIP DECK forcing suite please see the [CMIP Forcings Task Team webpage](https://wcrp-cmip.org/cmip7-task-teams/forcings/), the recent open community drop-in session [slides](https://wcrp-cmip.org/wp-content/uploads/2024/10/2024-10-29_Forcings-drop-in_session.pdf) and [recording](https://esait.webex.com/esait/ldr.php?RCID=19a0cdb6fd9d5ada593958dfb1a9daf2), and the input4MIPs documentation repository (https://input4mips-controlled-vocabularies-cvs.readthedocs.io/en/latest/dataset-overviews/) 

All CMIP6Plus era data sets are for testing purposes only, i.e. they are not to be relied upon for CMIP7 production runs. CMIP7 datasets to support the AR7 Fast Track will be made available in early 2025.

**Comment by DirkOlivie** [comment]
Forcing data set CMIP6Plus can now be found on nird in the directory :
/datalake/NS9560K/noresm3/input4mips/cmip6Plus-20241122

It contains the following forcings :
1. Anthropogenic short-lived climate forcer (SLCF) and CO2 emissions (CEDS-CMIP-2024-10-21), 222 files.  Currently no SO2, NOx, N2O and CO2 emissions are available (have been retracted).
2. Open biomass burning emissions (DRES-CMIP-BB4CMIP7-1-0), 298 files.
3. Land use (UofMD-landState-3-0), 4 files.
4. Greenhouse gas concentrations (CR-CMIP-0-3-0), 230 files.
5. Stratospheric volcanic SO2 emissions and aerosol optical properties (UOEXETER-CMIP-1-1-3), 6 files.
6. Solar forcing (SOLARIS-HEPPA-CMIP-4-4), 3 files.
7. AMIP sea-surface temperature and sea-ice boundary forcing (CMIP_PCMDI-AMIP-1-1-9), 6 files.
8. OMIP forcing(OMIP_MRI-JRA55-do-1-6-0), 224 files.

**Comment by DirkOlivie** [comment]
Overview of available input4MIPs data sets and their identifiers.  Some data sets are only in their testing phase (italic).

|    |                                                       | DECK (piControl+historical) | Scenarios |
|---|--------------------------------------------------------------------------------------------|-------------------------------------------------------|------------|
| 1 | Anthropogenic short-lived climate forcer (SLCF) and CO2 emissions | CEDS-CMIP-2025-04-18, CEDS-CMIP-2025-04-18-supplemental | _IIASA-IAMC-scendraft1-0-3-0, IIASA-IAMC-scendraft2-0-3-0_ |
| 2 | Open biomass burning emissions | DRES-CMIP-BB4CMIP7-2-0 | _IIASA-IAMC-scendraft1-0-3-0, IIASA-IAMC-scendraft2-0-3-0_ |
| 3 |Land use | UofMD-landState-3-1-1 |  | 
| 4 | Greenhouse gas concentrations | CR-CMIP-1-0-0 | _CR-vllo-0-1-0, CR-vlho-0-1-0, CR-l-0-1-0, CR-ml-0-1-0, CR-m-0-1-0, CR-hl-0-1-0, CR-h-0-1-0_ |
| 5 | CO2 isotopes | ImperialCollege-3-0 |  |
| 6 | Stratospheric volcanic SO2 emissions and aerosol optical properties | UOEXETER-CMIP-2-2-1 | UOEXETER-ScenarioMIP-2-2-1 |
| 7 | Ozone concentrations | FZJ-CMIP-ozone-1-2 |  |
| 8| Nitrogen deposition | FZJ-CMIP-nitrogen-1-2 |   |
| 9 |Solar | SOLARIS-HEPPA-CMIP-4-6 |  |
| 10 | AMIP sea-surface temperature and sea-ice boundary forcing | PCMDI-AMIP-1-1-10 |  
| 11 | Aerosol optical properties/MACv2-SP |   |  |  
| 12 | Population density (?) | PIK-CMIP-1-0-1 | PIK-vl-1-0-0, PIK-ln-1-0-0, PIK-l-1-0-0, PIK-ml-1-0-0, PIK-m-1-0-0, PIK-hl-1-0-0, PIK-h-1-0-0 |

**Comment by DirkOlivie** [comment]
Overview of forcing data needed by NorESM3.

| Component  | Data set                                   | Processing/preparation of data set               |
|------------------|-----------------------------------------|-----------------------------------------------------------|
| Atmosphere | Aerosol emissions                   | (Dirk)                                                             |
|                     | GHG concentrations                | (Dirk)                                                             |
|                     | Solar                                        | Based on input4MIPs, processed for CESM |  
|                     | Stratospheric AOD                  | (Dirk)                                                             |
|                     | Ozone concentration               | Use CESM data set                                      |
|                     | Oxidant concentrations           | Use CESM data set                                       |
|                     | Stratospheric H2O emissions | Use CESM data set                                        |
|                     |                                                 |                                                                        |
| Land            | Land use                                 | (Kjetil)                                                             |
|                     |                                                 |                                                                       |
| Ocean         | N-deposition                            | (Jörg, Mariana)                                              |

**Comment by DirkOlivie** [comment]
Some specifications on the processing of the various forcing data sets.

Forcings prepared by NorESM team :
|                                       | Grid                                                                                                          | Processing                                             |
|--------------------------------|---------------------------------------------------------------------------------------------|-------------------------------------------------------|
| **Forcings prepared by NorESM team** | |                                       
| GHG concentrations     | input4MIPS (15 degree latitude) -> 0.5 degree latitude (as in NorESM2) |                                                              |
| Aerosol emissions        | input4MIPs (0.25 and 0.5 degree) ->  ne16pg3, ne30pg3, fv19 (for NorESM2 and NorESM2.3), fv09 (for NorESM2 and NorESM2.3) | additional tropospheric volcanic outgassing; partitioning among different aersol modes; correction for topography; vertical profile for BB, volcanic and some anthrop. emissions |
|                                      |                                                         |                                                                |
| Volcanic AOD               | input4MIPs (5 degree latitude) + interpolate on-line                              | Interpolate from 47 to 14 (SW) and 16 LW wavelength bands |                                                               |
| CO2 emissions             |                                                                                                                  |                                                                |
| N-deposition                 | Using +/- original input4MIPs files (2x2.5 degrees) + interpolate on-line |                                                                |
| Land use                      | ne16pg3                                                                                                    |                                                                |
|                                      | ne30pg3                                                                                                    |                                                               |
|                                      |                                                                                                                   |                                                               |
| **Forcings hopefully using from CESM** |   |
| Ozone                          | CESM (mid-top, 93 levels)                                                                         |                                                               |
| Oxidants                      | CESM (mid-top, 93 levels)                                                                          |                                                              |
| H2O emissions            | CESM (WACCM CMIP6, 70 lelves / mid-top, 93 levels)                             |                                                              |

**Comment by DirkOlivie** [comment]
Issues to be discussed/decided:
1. Who prepares the CO2 emission files?
2. Do we keep the aerosol-scheme without explicit GHGs?  Or do we switch to a explicitly modelling the GHGs?  In the latter case, we do not need the H2O emission files anyore. However, we have to find out how the CO2-emission driven simulation can deal with these new additional CO2 tracer.

**Reply by oyvindseland** [reply]
The brute force (and simplest?) solution may be to define this CO2 tracer with another name and then a namelist variable defining which tracer should be used for radiation.

**Comment by JorgSchwinger** [comment]
> 2. Do we keep the aerosol-scheme without explicit GHGs? Or do we switch to a explicitly modelling the GHGs? In the latter case, we do not need the H2O emission files anyore. However, we have to find out how the CO2-emission driven simulation can deal with these new additional CO2 tracer.

Maybe you could say a bit more about this issue on the forcings-meeting on Thursday? Difficult to judge without further details

**Comment by DirkOlivie** [comment]
@adagj @tjiputra I have made GHG concentration forcing files for abrupt-4xCO2 and 1pcCO2.  They are both on betzy and olivia

Betzy directory : `/cluster/shared/noresm/inputdata/atm/waccm/lb/cmip7_ghg_version20260607`
Olivia directory : `/cluster/work/projects/nn9560k/inputdata/atm/waccm/lb/cmip7_ghg_version20260607`

abrupt-4xCO2 file : `ghg_cmip7_CMIP-abrupt4xco2_CR-CMIP-1-0-0_1850_monthly_gnz_0p5degLat_version20260207.nc`
1pctCO2 file : `ghg_cmip7_CMIP-1pctco2_CR-CMIP-1-0-0_0000-0166_monthly_gnz_0p5degLat_version20260207.nc`

The 1pctCO2 file contains the years 0 until 166.  Year 0 has preindustrial GHG concentrations, and the increase in CO2 starts from January in year 1.
